### ISSUE TITLE
Feature Request: Disable feed

### ISSUE THREAD

**Issue by distrob** [Ranchero-Software/NetNewsWire#3401]
Details: I'd like to be able to manually disable some feeds and folders. Making so could grey them out or hide them as as read feeds. _Archived_ could also be configured to be a smart feed.

Why this request? I'm probably not alone in taking time to curate my feeds and categorize them. But there are times in which certain feeds are less relevant to one's life. Deleting them would be to drastic. A good example could be my _news_ category.  Whilst I find important to generally keep up with world news, there are moments in my life that I'd like to abstain from it. This could be a "detox" period, in which I want to not be exposed to certain topics.

Thanks

**Comment by numist** [comment]
I wanted this last week! I was replacing [Kill The Newsletter](https://kill-the-newsletter.com) with [Feed Your Email](https://feedyour.email) but wasn't ready to delete the old feeds yet—wound up moving them to "On My Mac" and making the whole account inactive but I can see how that might not work for everyone.

**Comment by dmshaw** [comment]
I would love this ability. Deleting feeds when I don't want them and re-adding them when I do is the obvious workaround, but of course means any items I have starred disappear as well, so it's not a particularly good workaround.

**Comment by numist** [comment]
I assume deleting a feed also deletes any starred posts from that feed that might have rolled off the source? Seems destructive as a workaround.

**Comment by dmshaw** [comment]
Yes, exactly: any starred items disappear, including those that don't exist on the source any longer.  Like I said, it is not a particularly good workaround.

**Comment by jonhendry** [comment]
I'd like to see this as well.

I just added a feed for "rpilocator.com" which posts alerts when a retailer gets Raspberry Pis in stock. It posts frequently. I'm not actually looking for a Pi right now, so the posts clutter up NetNewsWire, but I don't want to delete the feed because I might need it later.

**Comment by djpowers** [comment]
I'd love to see this ability as well.

As another use case, I've found a handful of job board RSS feeds that are certainly useful when searching for work, but not as necessary when you are not actively looking. Disabling would be a great way to silence the noise but not lose reference for when you'd like to reactivate them.

**Comment by 707** [comment]
A year later, but also came here looking for a mute option. I like NNW is great for it's simplicity but a simple temporary mute option for x days would be convenient.
